src/main/resources/citizen/templates/document/defendantResponseReceipt.html

```diff
@@ -277,6 +281,29 @@
           {{ responseDefence }}
         </td>
       </tr>
+      {% if claim.events is defined %}
+      <tr class="no-border">
+        <td colspan="2" style="white-space: pre-line"><strong>Defendant's timeline of what happened:</strong></td>
+      </tr>
+      {% for event in events %}
+      <tr class="no-border">
+        <td>
+          {{ event.date | trim }}
+        </td>
+        <td>
+          {{ event.description | trim }}
+        </td>
+      </tr>
+      {% endfor %}
+      {% endif %}
+
+      <tr class="no-border">
+        <td><strong>Comments about claimant's timeline:</strong></td>
+        <td>
+          {{ timelineComment }}
```

Comment:
check what should go here if not provided

Reply:
done